app.js

```diff
@@ -38,113 +38,111 @@ var Storage = multer.diskStorage({
+            var sql = "INSERT INTO speaker(name, topic, description, pic_url) " +
+                        "values('"+name+"','"+topic+"','"+description+"','/images/"+name+".jpg')";
+            con.query(sql, function (err, result, fields) {
+                res.redirect('/admin');
+            });
         }
-        insertSpeaker(req , res);
     });
 });
 app.get('/logout', function (req, res) {
     req.session.destroy();
     res.redirect("/login");
 });
 app.post('/getuser' , function (req , res) {
-    res.send({"username" : req.session.user});
+    res.send({
+        "username" : req.session.username
+    });
 });
 app.get('/blog' , function (req , res) { 
 	res.sendFile(__dirname + '/www/blog.html');
 });
 app.post('/blog' , function (req , res) { 
     var sql = "SELECT * FROM videos";
     con.query(sql, function (err, result, fields) {
-        var jsonString = JSON.stringify(result);
-        var jsonData = JSON.parse(jsonString);
-        res.send(jsonData);
+        res.send(result);
     });
 });
-app.post("/videoinsert", function(req, res) {
-    insertVideo(req , res);
-});
-function login(req,res){
-    var post = req.body;
-    var username  = post.user;
-    var password = post.password;
-    username = sanitizeHtml(username);
-    password = sanitizeHtml(password);
-    var sql = "SELECT * FROM login WHERE username='"+username+"'";
+app.post("/video-insert", function(req, res) {
+    const post = req.body;
+    const title  = sanitizeHtml(post.title);
+    const description = sanitizeHtml(post.description);
+    const url = sanitizeHtml(post.url);
+    var sql = "INSERT INTO videos(title, description, video_url) " +
+                "values('"+title+"','"+description+"','"+url+"')";
     con.query(sql, function (err, result, fields) {
-        var jsonString = JSON.stringify(result);
-        var jsonData = JSON.parse(jsonString);
-        if( jsonData.length > 0 && jsonData[0].password === password ) {
-            console.log(req.session.user + "Auth set");
-            req.session.user = post.user;
-            res.send({"result" : "Found"});
-        }else {
-            res.send({"result": "NotFound"});
-        }
+        res.redirect('/admin');
     });
-}
-function insertSpeaker(req , res ) {
-    var post = req.body;
-    var name  = post.speakername;
-    var topic = post.topic;
-    var description = post.description;
-    name = sanitizeHtml(name);
-    topic = sanitizeHtml(topic);
-    description = sanitizeHtml(description);
-    var sql = "INSERT INTO speaker(name, topic, description, pic_url) values('"+name+"','"+topic+"','"+description+"','/images/"+name+".jpg')";
+});
+app.post("/delete-speaker", function(req, res) {
+    const post = req.body;
+    const speakerName  = sanitizeHtml(post.speakerName);
+    var sql = "Delete from speaker where name = '" + speakerName +"'" ;
     con.query(sql, function (err, result, fields) {
         res.redirect('/admin');
```

Comment:
Stick with one convention, either double or single quotes.

A linter will be really helpful.